ghga_transpiler/core/__init__.py

```diff
@@ -13,7 +13,5 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-"""
-This sub-package contains the main business functionality of this service.
-It should not contain any service API-related code.
-"""
+"""Module to load core functions for workbook processing"""
+from .core import convert_rows, get_header, get_worksheet_rows, read_workbook  # noqa
```

Comment:
```suggestion
from .core import (  # noqa: F401
    convert_rows,
    get_header,
    get_worksheet_rows,
    read_workbook,
)
```
Be specific which error you want the linter to ignore, in this case [Module imported but unused](https://www.flake8rules.com/rules/F401.html).